django_freeradius/tests/test_admin.py

```diff
@@ -213,3 +213,15 @@ def test_nas_admin_save_model(self):
         self.assertNotContains(response, 'error')
         nas.refresh_from_db()
         self.assertEqual(nas.type, 'my-custom-type')
+
+    def test_cleanup_stale_radacct_command(self):
+        RadiusAccounting.objects.create(
+            unique_id='117', username='bob', nas_ip_address='127.0.0.1', start_time='2017-06-10 10:50:00',
+            authentication='RADIUS', connection_info_start='f', connection_info_stop='hgh',
+            input_octets='1', output_octets='4', update_time='2017-03-10 11:50:00'
```

Comment:
I suppose some of these attributes are shared between the other tests that you created previously, if that's the case please consider creating a dictionary which is reused, eg:

```python
options = _RADACCT.copy()
options['username'] = 'example-of-how-to-change-an-attribute'
RadiusAccounting.objects.create(**options)
```